docs/AggregateTranslations.md

```diff
@@ -23,6 +23,7 @@ Once all of the translation files are created for the specified locales, the scr
 | locales  | -l, --locales | Array of Strings | The list of locale codes to aggregate. **Note: 'en' is always added if not specified.** | [terra-supported locales](https://github.com/cerner/terra-core/blob/master/packages/terra-i18n/src/i18nSupportedLocales.js) |
 | outputDir | -o, --ouputDir | String | Output directory for the translation and loader files | ./aggregated-translations |
 | configPath | -c, --config | String | The path to the terra i18n configuration file | undefined |
+| format | -f, --format | String | The format of syntax to output the translations with. Setting to undefined outputs the syntax with ES5 format. Setting format to 'modern' outputs the translations with ES6 modern syntax | undefined |
```

Comment:
I am wondering if it would clearer to use an option called `esVersion` or `esSyntax` for when new es versions are released and if the syntax does change. Modern makes sense at the moment, but won't if something changes later on.

Reply:
I like this, I'm thinking we can keep the `format` flag and instead of passing `modern` we pass `es6`. I also want to maintain the option to not pass a format flag and have it default to es5 for passivity sake.